KantoBot.py

```diff
@@ -71,7 +75,7 @@ def __init__(self, bot):
 
     print("Creating kantobot info for user " + user.name)
 
-    # Create the base trainer role iff it doesn't exist
+    # Create the base trainer role if it doesn't exist
```

Comment:
This is intentional. Iff, with the double f, means [if and only if](https://en.wikipedia.org/wiki/If_and_only_if)